src/generator.cpp

```diff
@@ -32,6 +32,14 @@ limitations under the License.
 #include "generatorprofiletools.h"
 #include "utilities.h"
 
+#ifdef TRUE
+#    undef TRUE
+#endif
+
+#ifdef FALSE
+#    undef FALSE
+#endif
+
 #ifdef NAN
 #    undef NAN
 #endif
```

Comment:
Does anyone else think these need an explanation?

Reply:
It has to do with building on Linux. I could confirm (same with `NAN`) and make a comment about it, if really needed.